src/main/java/com/launchdarkly/eventsource/AsyncEventHandler.java

```diff
@@ -27,6 +27,18 @@ public void run() {
     });
   }
 
+  public void onComment(final String comment) {
+    executor.execute(new Runnable() {
+      public void run() {
+        try {
+          eventSourceHandler.onComment(comment);
+        } catch (Throwable ignored) {
+
```

Comment:
Can you explain why this exception is silently ignored? Would it be good to add a debug log statement containing the exception?